ndohyep/views.py

```diff
@@ -0,0 +1,67 @@
+from django.views.generic.base import TemplateView
+from django.conf import settings
+
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import get_object_or_404
+from django.shortcuts import render
+
+from molo.core.models import ArticlePage
+from wagtail.wagtailsearch.models import Query
+
+import django_comments
+
+from molo.commenting.forms import MoloCommentForm
+from molo.commenting.models import MoloComment
+
+
+class CommentReplyForm(TemplateView):
```

Comment:
Hmm...this should actually be called `CommentReplyFormView` - otherwise it can be confused with a Form